scripts/appleseedMaya/renderGlobals.py

```diff
@@ -472,32 +479,39 @@ def create(self):
                         limitBounces = mc.getAttr(
                             "appleseedRenderGlobals.limitBounces")
                         self._addControl(
-                            ui=pm.intFieldGrp(
-                                label="Global Bounces", numberOfFields=1, enable=limitBounces),
+                            ui=pm.intSliderGrp(
+                                label="Global Bounces", field=True, value=8, cw=(3,160), minValue=0, 
```

Comment:
I wonder if maxValue should be set to 100 as well? I would tend to think so (for all bounce types).

Reply:
The slider has 30 as soft limit but one can override it up to 100 in the numeric field. 
If the slider ranges are large ( goes up to 100) then a small movement of the slider and the bounces change by 5 or 10, which makes it difficult to adjust it for most purposes. 30 bounces is quite large for most scenes.